saraho1123/whats-for-dinner
# Feature/see recipes button

## Is this a fix or a feature?  
feature

## What is the change? 
Added a "See Recipes!" button that takes the string of the side, main dish or dessert being displayed in the "You should make" suggestion box and uses it to search all recipes website. 

## What does it fix? 
n/a

## Where should the reviewer start? 
line 45 of index.html
line 61 of data.js
line 8 of main.js
line 1 of query-and-event-listeners.js

## How should this be tested?
- open index.html
- choose side, main dish, or dessert
- See Recipes! button will appear at bottom left of food suggestion (button will not appear for entire meal)

- also from the Add Recipe form, when the Add New button is clicked, the See Recipes! button will appear

- When the See Recipes! button is clicked, it will open a new tab on the All Recipes website, using whatever food was being suggested in the search. Recipe suggestions for that food should appear.

### Screenshots (if applicable):



@sarah01123
